README.md

```diff
@@ -13,7 +15,52 @@ Used together, you get a powerful tool for developing high-performance angular a
 
 This repository holds a set of helpers to create **fully reactive** as well as **fully zone-less** applications.
 
-[![rx-angular logo](https://raw.githubusercontent.com/rx-angular/rx-angular/main/docs/images/rx-angular_logo.png)](https://www.rx-angular.io/)
+## Benefits
+
+ - ✔ Migration scripts form beat versions on! Just run `ng update @rx-angular/???`
+ - ✔ No boilerplate garantieed
+ - ✔ Performance focused and small bundle size through sub packages
+ - ✔ Well typed and tested
+ - ✔ support for Angular > v11
+ - ❗ expect no migration scripts for changes in `experimental` folders
```

Comment:
```suggestion
 - 🔥 It's fast & performance focused: exceptional runtime speed & small bundle size
 - ✔ Easy upgrade paths: migration scripts included since beta! `ng update @rx-angular/{cdk | template | state}`
 - ✔ Lean and simple: No boilerplate guaranteed
 - ✔ Well typed and tested
 - ✔ Backwards compatible: support for Angular > v11
```

Reply:
> - ❗ expect no migration scripts for changes in `experimental` folders

The reason fot 1 exclamation mark was to put the rest in contrast. A small trick from marketing. "Only one thing next to all the good stuff. 🤯"

Reply:
I like the reordering! Definitely better now.

Reply:
sorry, but having no migration scripts is no 'benefit' thats why i removed it